web/react/components/rhs_comment.jsx

```diff
@@ -67,6 +68,9 @@ class RhsComment extends React.Component {
     parseEmojis() {
         twemoji.parse(ReactDOM.findDOMNode(this), {size: Constants.EMOJI_SIZE});
     }
+    handlePermalink() {
+        EventHelpers.showGetPostLinkModal(this.props.post);
```

Comment:
Can you add an argument `e` and add `e.preventDefault()` at the beginning of this function? Otherwise a # will be added to the URL address bar when ever the link is clicked